htlc/package.json

```diff
@@ -1,6 +1,71 @@
 {
   "name": "htlc-dapp",
   "version": "1.0.0",
-  "main": "index.js",
-  "license": "MIT"
+  "private": true,
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "chai": "^4.2.0",
+    "customize-cra": "^1.0.0",
+    "eslint-config-prettier": "^8.1.0",
+    "eslint-plugin-react": "^7.22.0",
+    "eslint-plugin-simple-import-sort": "^7.0.0",
+    "eslint-plugin-sonarjs": "^0.6.0",
+    "husky": "4.3.0",
+    "lint-staged": "^10.5.4",
+    "prettier": "^2.2.1",
+    "react-app-rewire-yaml": "^1.1.0",
+    "react-app-rewired": "^2.1.8"
+  },
+  "dependencies": {
+    "@algorand-builder/algob": "^0.5.3",
```

Comment:
We can remove it and add `algob` as a script (same as `/default`). Or we can keep this as well (if you prefer this).

Reply:
Also please update package to `@algo-builder/algob`

Reply:
Okay